_schemaV2.graphql

```diff
@@ -76,6 +76,7 @@ type Agreement {
 
 type Algolia {
   apiKey: String
+  appID: String
```

Comment:
**observation:** noticed that inside this file in most of the cases we use camelCase when having id as the second part of the string i.e. `partnerId` vs `partnerID` although both looks good to me

Reply:
I think most of our V2 schema leans toward `somethingID` naming. These schema design [docs](https://github.com/artsy/README/blob/main/playbooks/graphql-schema-design.md#id-fields) in artsy/README mention IDs of objects in our database but kind of applies here as well.

Reply:
Sounds good! feel free to merge whenever